### PR TITLE
Add tests for Cloud SQL samples

### PR DESCRIPTION
**Pull request by shubha-rajan**
- [X] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [X] `pom.xml` parent set to latest `shared-configuration`
- [X] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Comment by shubha-rajan** [comment]
cc: @kurtisvg @craiglabenz

File: .kokoro/tests/run_tests.sh [hunk 1]
```diff
@@ -78,6 +78,8 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
     source "${KOKORO_GFILE_DIR}/firestore_secrets.txt"
     # spellcheck source=src/cts_v4_secrets.txt
     source "${KOKORO_GFILE_DIR}/cts_v4_secrets.txt"
+    # shellcheck source=src/cloud_sql_secrets.txt
+    source "${KOKORO_GFILE_DIR}/cloud_sql_secrets.txt"
```
Comment (averikitsch) [inline .kokoro/tests/run_tests.sh:82]: Have you already set this up? If not we can use secret manager instead.

Comment (shubha-rajan) [reply .kokoro/tests/run_tests.sh:82]: I have but it's not working, so let's try using Secret Manager instead

Comment (averikitsch) [reply .kokoro/tests/run_tests.sh:82]: sgtm. Dropping this example if you decided to switch https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/.kokoro/build-with-run.sh#L92

Comment (shubha-rajan) [reply .kokoro/tests/run_tests.sh:82]: Got it working using the secrets file in Cloud Devrel Kokoro Resources. I think for now we can keep this setup, but eventually we should start migrating all the secrets to secret manager.

File: cloud-sql/mysql/servlet/src/test/java/com/TestIndexServlet.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2018 Google LLC
```
Comment (averikitsch) [inline cloud-sql/mysql/servlet/src/test/java/com/TestIndexServlet.java:2]: update year for new files

**Review by averikitsch — APPROVED** [review]

**Review by kurtisvg — CHANGES_REQUESTED** [review]

File: cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2018 Google LLC
```
Comment (eaball35) [inline cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java:2]: Wrong year

Comment (eaball35) [inline cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java:2]: Wrong year

File: cloud-sql/mysql/servlet/src/test/java/com/TestIndexServlet.java [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    HikariConfig config = new HikariConfig();
+
+    config.setJdbcUrl(String.format("jdbc:mysql:///%s", System.getenv("MYSQL_DB")));
+    config.setUsername(System.getenv("MYSQL_USER")); // e.g. "root", "mysql"
+    config.setPassword(System.getenv("MYSQL_PASS")); // e.g. "my-password"
+    config.addDataSourceProperty("socketFactory", "com.google.cloud.sql.mysql.SocketFactory");
+    config.addDataSourceProperty("cloudSqlInstance", System.getenv("MYSQL_CONNECTION_NAME"));
+
+    pool = new HikariDataSource(config);
+    createTable(pool);
```
Comment (eaball35) [inline cloud-sql/mysql/servlet/src/test/java/com/TestIndexServlet.java:84]: Does this create a table that needs be to cleaned up after tests?

File: cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java (hunk 3 above)
Comment (eaball35) [inline cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java:2]: old year

File: cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+import java.util.Map;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+public class TestIndexServlet {
```
Comment (eaball35) [inline cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java:42]: Can name be more specific? I thought all of these were duplicates until I looked closer.

**Review by kurtisvg — APPROVED** [review]

File: cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java (hunk 5 above)
Comment (shubha-rajan) [reply cloud-sql/postgres/servlet/src/test/java/com/TestIndexServlet.java]: Renamed these to include the database engine.

File: cloud-sql/mysql/servlet/src/test/java/com/TestIndexServlet.java (hunk 4 above)
Comment (shubha-rajan) [reply cloud-sql/mysql/servlet/src/test/java/com/TestIndexServlet.java]: added a cleanup function

**Review by eaball35 — APPROVED** [review]

**Pull request merged**